charts/ingress-nginx/templates/controller-ingressclass.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.controller.ingressClass.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  labels:
+    {{- include "ingress-nginx.labels" . | nindent 4 }}
+    app.kubernetes.io/component: controller
+    {{- with .Values.controller.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ include "ingress-nginx.controller.fullname" . }}
```

Comment:
I think the value here is wrong, shouldnt metadata.name point to the name of the ingressclass, aka. `.Values.controller.ingressClass.name` ?

Reply:
This makes sense yes